generator.ts

```diff
@@ -0,0 +1,220 @@
+import { Confirm, Input } from "@webpack-cli/webpack-scaffold";
+import Generator from "yeoman-generator";
+import createDevConfig from "./dev-config";
+import questions from "./questions";
+import getPackageManager from "./utils/package-manager";
+import welcomeMessage from "./utils/welcome";
+
+module.exports = class WebpackGenerator extends Generator {
+
+ [x: string]: any;
+ public dependencies: string[];
+ public devConfig: any;
+ public webpackOptions: any;
```

Comment:
There is no use in specifying `any` here. We can just create an interface that will help us a lot. A future enhancement for sure 👍